packages/api/src/user.js

```diff
@@ -100,6 +100,7 @@ export function withAuth (handler) {
    */
   return async (request, env, ctx) => {
     const auth = request.headers.get('Authorization') || ''
+    // TODO: Should this throw if no auth token with meaningful error?
```

Comment:
Currently sending empty Authotization seems to throw a magic link error. Should we make this more friendly?

Reply:
We do some of this in nft.storage https://github.com/ipfs-shipyard/nft.storage/blob/aad125902fe4f8e6b5144ed52e60be3988f24e1a/packages/api/src/errors.js#L41-L68

Reply:
ok, so should I create an issue to get that into here?

Reply:
yes good plan